pkg/config/logger.go

```diff
@@ -0,0 +1,23 @@
+package config
+
+import (
+	"io"
+	"log/syslog"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+func GetLogger(verbose bool) (*logrus.Logger, error) {
+	var log = logrus.New()
+	sysWriter, _ := syslog.New(syslog.LOG_INFO, "sentinel")
```

Comment:
Why ignoring the error and not returning it?

Reply:
Yeah it's dumb since I am testing the error when creating the logger.
Will fix that thanks.